[2024-03-26 06:42:23,434] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.validate_data 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2 [queued]>
[2024-03-26 06:42:23,443] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: bigdag.validate_data 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2 [queued]>
[2024-03-26 06:42:23,443] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 06:42:23,443] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-03-26 06:42:23,444] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-03-26 06:42:23,460] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): validate_data> on 2024-03-26 06:42:10.655925+00:00
[2024-03-26 06:42:23,469] {standard_task_runner.py:52} INFO - Started process 2093 to run task
[2024-03-26 06:42:23,474] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'bigdag', 'validate_data', '23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2', '--job-id', '1361', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpg12srq6h', '--error-file', '/tmp/tmplefm62dp']
[2024-03-26 06:42:23,476] {standard_task_runner.py:80} INFO - Job 1361: Subtask validate_data
[2024-03-26 06:42:23,598] {task_command.py:369} INFO - Running <TaskInstance: bigdag.validate_data 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2 [running]> on host 911e7a0235cb
[2024-03-26 06:42:23,749] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=bigdag
AIRFLOW_CTX_TASK_ID=validate_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-26T06:42:10.655925+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2
[2024-03-26 06:42:23,755] {logging_mixin.py:115} INFO - {'conf': <***.configuration.AirflowConfigParser object at 0xffff84fe5450>, 'dag': <DAG: bigdag>, 'dag_run': <DagRun bigdag @ 2024-03-26 06:42:10.655925+00:00: 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2, externally triggered: True>, 'data_interval_end': DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')), 'data_interval_start': DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')), 'ds': '2024-03-26', 'ds_nodash': '20240326', 'execution_date': <Proxy at 0xffff64d8c500 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'execution_date', DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')))>, 'inlets': [], 'logical_date': DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')), 'macros': <module '***.macros' from '/home/***/.local/lib/python3.7/site-packages/***/macros/__init__.py'>, 'next_ds': <Proxy at 0xffff64d8c550 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'next_ds', '2024-03-26')>, 'next_ds_nodash': <Proxy at 0xffff64d8c5a0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'next_ds_nodash', '20240326')>, 'next_execution_date': <Proxy at 0xffff64d8c5f0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'next_execution_date', DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')))>, 'outlets': [], 'params': {'s3_uri': 's3://cfainstitute-learning-outcomes-raw/t1/Level2_combined.pdf'}, 'prev_data_interval_start_success': DateTime(2024, 3, 26, 6, 38, 43, 963540, tzinfo=Timezone('UTC')), 'prev_data_interval_end_success': DateTime(2024, 3, 26, 6, 38, 43, 963540, tzinfo=Timezone('UTC')), 'prev_ds': <Proxy at 0xffff64d8c640 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'prev_ds', '2024-03-26')>, 'prev_ds_nodash': <Proxy at 0xffff64d8c690 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'prev_ds_nodash', '20240326')>, 'prev_execution_date': <Proxy at 0xffff64d8c6e0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'prev_execution_date', DateTime(2024, 3, 26, 6, 42, 10, 655925, tzinfo=Timezone('UTC')))>, 'prev_execution_date_success': <Proxy at 0xffff64d8c730 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'prev_execution_date_success', DateTime(2024, 3, 26, 6, 38, 43, 963540, tzinfo=Timezone('UTC')))>, 'prev_start_date_success': DateTime(2024, 3, 26, 6, 38, 44, 245921, tzinfo=Timezone('UTC')), 'run_id': '23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2', 'task': <Task(PythonOperator): validate_data>, 'task_instance': <TaskInstance: bigdag.validate_data 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2 [running]>, 'task_instance_key_str': 'bigdag__validate_data__20240326', 'test_mode': False, 'ti': <TaskInstance: bigdag.validate_data 23a4b6a5-8d92-40ae-bc5e-c6985f0ff2a2 [running]>, 'tomorrow_ds': <Proxy at 0xffff64d8c780 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'tomorrow_ds', '2024-03-27')>, 'tomorrow_ds_nodash': <Proxy at 0xffff64d8c7d0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'tomorrow_ds_nodash', '20240327')>, 'ts': '2024-03-26T06:42:10.655925+00:00', 'ts_nodash': '20240326T064210', 'ts_nodash_with_tz': '20240326T064210.655925+0000', 'var': {'json': None, 'value': None}, 'conn': None, 'yesterday_ds': <Proxy at 0xffff64d8c820 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'yesterday_ds', '2024-03-25')>, 'yesterday_ds_nodash': <Proxy at 0xffff64d8c870 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0xffff64df6b90>, 'yesterday_ds_nodash', '20240325')>, 'templates_dict': None}
[2024-03-26 06:42:23,788] {logging_mixin.py:115} INFO - 1 validation error for LearningOutcomes
topic
  Input should be a valid string [type=string_type, input_value=nan, input_type=float]
    For further information visit https://errors.pydantic.dev/2.5/v/string_type
[2024-03-26 06:42:23,789] {python.py:173} INFO - Done. Returned value was: Level2_combined.csv
[2024-03-26 06:42:23,849] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=bigdag, task_id=validate_data, execution_date=20240326T064210, start_date=20240326T064223, end_date=20240326T064223
[2024-03-26 06:42:23,907] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-03-26 06:42:23,953] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
